main.html

```diff
@@ -1,77 +1,255 @@
+  It works for the producing team because they are able to better scale and serve their consumers.
+  It works for the company at-large because solutions to shared problems end up in shared, centrally-maintained locations where all can use them. 
+  Additionally, engineering focus stays on producing the code that solves company problems rather than the mechanics of the feature negotiation and escalation process.
+</p>
+
+<h2>Solution Detail</h2>
+<p>
+  Let's take a closer look at how the mechanics of InnerSource can work.
+  To aid in this explanation, we'll define a few key terms.
+  First, in the context of the code contribution process, we call the team that is receiving the code the <strong>Host team</strong>.
+  Conversely, the team giving the code is the <strong>Guest team</strong>.
+  There are also a few key individuals to name.
+  First, the <strong>Product Owner</strong> determines what functionality the host team is willing to accept as contributions.
+  The <strong>Contributor</strong> is the individual submitting the code contribution.
+  The <strong>Trusted Committer</strong> provides any timely support and mentorship that the contributor needs to successfully submit the code contribution.
+</p>
+
+<p>
+  With those definitions, here is the basic outline of an InnerSource contribution.
+</p>
+<p>
+  <ol>
+    <li>
+      Guest team requests a feature from the host team.
+    </li>
+    <li>
+      Product owner agrees that this feature is desirable in the host team's system.
+    </li> 
+    <li>
+      As the product owner prioritizes work for the host team, items other than the guest team's request are chosen.
+    </li>
+    <li>
+      Host team communicates the (de)prioritization of the guest team's request and asks if they would like to complete it via InnerSource contribution.
+    </li>
+    <li>
+      Guest team agrees to submit the feature request via InnerSource.
+    </li>
+    <li>
+      Product owner ensures that user stories representing the feature request are created.
+      These stories should describe the requested feature in terms agreeable to the guest team.
+      They also list any details from the host team on how the feature should be delivered in order for the work to be accepted.
+      Examples of such details include architecture constraints, coding conventions, dependency usages, data contracts, etc.
+    </li>
+    <li>
+      Trusted committers' time is made available to the guest team when the contributor is working.
+    </li>
+    <li>
+      Supported by the trusted committer(s), the contributor submits the pull request(s) to implement the requested feature.
+    </li>
+    <li>
+      Pull request is merged, and the new feature is completed.
+    </li>
+  </ol>
+</p>
+<p>
+  Note that these steps do not assume a specific system for the general organization of a team's time or priorities.
```

Comment:
Seems like there should be something in here about conflict resolution as well.  I know that this can come up and is what makes this part.

Reply:
I was trying to get at the fact that some basic conflict resolution needs to take place in order to remove some of the barriers to adoption.  Now that the solution detail I think this part is good as it is.